Modules/Remote/ParabolicMorphology.remote.cmake

```diff
@@ -5,5 +5,5 @@ itk_fetch_module(ParabolicMorphology
   http://www.insight-journal.org/browse/publication/228
   https://hdl.handle.net/1926/1370"
   GIT_REPOSITORY ${git_protocol}://github.com/InsightSoftwareConsortium/ITKParabolicMorphology.git
-  GIT_TAG d4bc69362c3e435ff5d97ed5bee0167199da69ac
+  GIT_TAG d2fc2474c5f7342824856c690ac6ec7b2721cf1e
```

Comment:
This should use 996cfee503279ddb47e4969ef07a31f498429fe0. Possibly also wait for the integration of [PR19](https://github.com/InsightSoftwareConsortium/ITKParabolicMorphology/pull/19).